indexers/contacts.index.json

```diff
@@ -74,119 +74,117 @@
         }
     },
     "mappings": {
-        "_doc": {
```

Comment:
https://www.elastic.co/guide/en/elasticsearch/reference/7.17/removal-of-types.html